unity/unity-embed-host/CoreCLRHost.cs

```diff
@@ -327,6 +327,14 @@ public static IntPtr unity_method_get_function_pointer([NativeCallbackType("Mono
         IntPtr field)
         => FieldInfo.GetFieldFromHandle(field.FieldHandleFromHandleIntPtr(), RuntimeTypeHandle.FromIntPtr(klass)).ToNativeRepresentation();
 
+    [return: NativeCallbackType("MonoObject*")]
+    public static IntPtr assembly_get_object(
+        [ManagedWrapperOptions(ManagedWrapperOptions.Exclude)] [NativeCallbackType("MonoDomain*")]
+        IntPtr domain,
+        [NativeCallbackType("MonoAssembly*")]
+        IntPtr assembly)
+        => assembly.AssemblyFromGCHandleIntPtr().ToNativeRepresentation();
```

Comment:
Hah! You were right this turned out to be pretty easy.

Reply:
Thanks for doing the leg work 😄